terragrunt/aws/api/iam.tf

```diff
@@ -0,0 +1,84 @@
+data "aws_iam_policy_document" "api_policies" {
+
+  statement {
+
+    effect = "Allow"
+
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
+    ]
+
+    resources = [
+      "arn:aws:logs:${var.region}:${var.account_id}:log-group:*"
+    ]
+  }
+
+  statement {
+
+    effect = "Allow"
+
+    actions = [
+      "states:ListStateMachines",
+      "states:ListActivities",
+      "states:CreateActivity",
+      "states:DescribeExecution",
+      "states:StartExecution",
+    ]
+
+    resources = [
+      "arn:aws:states:${var.region}:${var.account_id}:*"
+    ]
+  }
+
+  statement {
+
+    effect = "Allow"
+
+    actions = [
+      "events:PutTargets",
+      "events:PutRule",
+      "events:DescribeRule"
+    ]
+
+    resources = [
+      "arn:aws:events:${var.region}:${var.account_id}:*"
+    ]
+  }
```

Comment:
We can get rid of all of these policies.  The first `statement` for CloudWatch gets auto-created and the other two `statement`s are for things we won't be using.

Reply:
Done!